app/routes/pages/components/PageHierarchy.css

```diff
@@ -4,31 +4,59 @@
   display: flex;
   flex-direction: column;
 
-  /* border-left: 1px solid #ddd; */
+  @media (--mobile-device) {
+    margin-left: 1.5em;
+  }
 }
 
 .pageList {
   text-align: left;
   line-height: 18px;
-
-  & li {
-    margin-bottom: 10px;
-  }
 }
 
 .pageList a {
   font-size: 18px;
 }
 
-.header {
+.dropdownBtn {
+  text-transform: uppercase;
+  letter-spacing: 1.5px;
   display: flex;
   flex-direction: row;
   align-items: center;
-  font-weight: bold;
   font-size: var(--font-size-large);
+  font-family: Raleway;
```

Comment:
Isn't `Raleway` the default font for default button? As well as `text-transform: uppercase`.

Reply:
If I remember correctly, this was not the case that it was the correct font without this , but can try again.